### PR TITLE
[GR] bump GR version

### PR DESCRIPTION
**Pull request by jheinen**
depend on Qt6 instead of Qt5

### CODE REVIEW

File: G/GR/build_tarballs.jl [hunk 1]
```diff
@@ -102,16 +95,16 @@ dependencies = [
     Dependency("GLFW_jll"),
     Dependency("JpegTurbo_jll"),
     Dependency("libpng_jll"),
-    Dependency("Libtiff_jll"; compat="~4.3, ~4.4"),
+    Dependency("Libtiff_jll"),
     Dependency("Pixman_jll"),
-#    Dependency("Qhull_jll"),
-    Dependency("Qt5Base_jll"),
+    HostBuildDependency("Qt6Base_jll"),
+    Dependency("Qt6Base_jll"),
```
Comment (barche) [inline G/GR/build_tarballs.jl:101]: Best to introduce a compat here:

```julia
Dependency("Qt6Base_jll"; compat="6.4.2"),
```

File: G/GR/build_tarballs.jl [hunk 2]
```diff
@@ -102,16 +95,16 @@ dependencies = [
     Dependency("GLFW_jll"),
     Dependency("JpegTurbo_jll"),
     Dependency("libpng_jll"),
-    Dependency("Libtiff_jll"; compat="~4.3, ~4.4"),
+    Dependency("Libtiff_jll"),
```
Comment (giordano) [inline G/GR/build_tarballs.jl:98]: Please do not remove compat bounds.  This one should actually be updated to use 4.5.1
```suggestion
    Dependency("Libtiff_jll"; compat="~4.5.1"),
```

**Review by barche — APPROVED** [review]
Looks good to me!

**Comment by jheinen** [comment]
@giordano : Are there things that need to be done before the PR can be merged?

**Pull request merged**